stellar/operators.js

```diff
@@ -10,20 +10,57 @@ async function processCommand(options, _, chain) {
 
     const contract = new Contract(options.address || chain.contracts?.axelar_operators?.address);
 
-    let operation;
+    let operator, operation;
 
-    const address = Address.fromString(options.args || wallet.publicKey());
+    if (['is_operator', 'add_operator', 'remove_operator'].includes(options.action)) {
+        if (!options.args) throw new Error(`Missing --args operatorAddress the params.`);
+        operator = Address.fromString(options.args).toScVal();
+    } else {
+        operator = Address.fromString(wallet.publicKey()).toScVal();
+    }
```

Comment:
this logic seems to be making it more complicated, since this is action specific logic. Maybe do the `address` to `operator` rename, but keep the `operator.toScVal()` below as before. you can add a validateOptions method that has action specific validation of `options`. The first 3 cases can fallthrough to the same validation check in the switch statement?